Makefile

```diff
@@ -1,18 +1,20 @@
-CC=gcc
-CFLAGS=-m32 -O3
-INSTALLDIR=/usr/bin
-BINPATH=bootstrap/mu
+ARCH?=32
+CC?=gcc
+CFLAGS=-m$(ARCH) -O3
+PREFIX?=/usr/local
```

Comment:
I'm not super familiar with bin directory conventions on Linux/Unix, so I'm hoping you can enlighten me :). Do you know what bin dir Linux generally uses? Is this distro-dependent? This change modifies the default install dir from /usr/bin to /usr/local/bin, is that something that we should do? Do you know if there's an existing variable that we can use to learn the default bin dir is for the user's OS?

Reply:
Here is some documentation about[ path hierarchy](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s09.html) in unix like systems.